### ISSUE TITLE
[BUG]:Getting an error - "Trying to heal... [main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request. java.io.IOException: unexpected end of stream on http://localhost:7878/..."

### ISSUE THREAD

**Issue by bjha1** [healenium/healenium-web#217]
### Describe the bug

getting an error -
"Trying to heal...
[main] WARN com.epam.healenium.client.RestClient - Failed to make response of 'getLastHealingData' request.
java.io.IOException: unexpected end of stream on http://localhost:7878/..."
<img width="1343" alt="Screenshot 2023-01-15 at 4 45 01 AM" src="https://user-images.githubusercontent.com/118807278/212533899-04564da6-33cf-410c-ad3b-dd0bef9f76db.png">
<img width="810" alt="Screenshot 2023-01-15 at 4 45 26 AM" src="https://user-images.githubusercontent.com/118807278/212533938-1f859ad0-fc15-4199-9bc0-857c1e89ed2e.png">

DOCKER IMAGE

### How to reproduce the issue

```shell
CREATE CONTAINER AND RUN TEST CASES
```

### Logs appeared during using Healenium

```shell
Failed to make response of 'getLastHealingData' request.
java.io.IOException: unexpected end of stream on http://localhost:7878/..."
```

### Healenium Web version

3.3.1

### Healenium Backend version

3.2.5

### Platform

JAVA

**Comment by bjha1** [comment]
@ElenaStepuro any update ?

**Comment by JackPieter** [comment]
hi, team I have the same problem
![image](https://user-images.githubusercontent.com/125430806/218950170-a0be7bc2-b394-4510-a1a3-b3f7b43512d8.png)

**Comment by JackPieter** [comment]
> @ElenaStepuro any update ?
This issues was solved when I started the clone backend service
I think maybe 'healenium/hlm-backend:3.2.5' is the problem.

**Comment by Alex-Reif** [comment]
HI @JackPieter , @bjha1 , We'll fix it in the next release

**Comment by Alex-Reif** [comment]
Hi @bjha1 , the issue was fixed at healenium-web 3.4.3
